LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2010 by Armin Ronacher.
+Copyright (c) 2016 by Ethan Miller.
```

Comment:
It's custom to still mention the original author. As in "based on the Flask-Babel Package by Armin Ronacher".